index.js

```diff
@@ -69,11 +69,14 @@ exports.handler = ({ body }, lambdaContext, callback) => {
           .then(() => githubRequest(setStatus(context, 'success'), context.userToken))
           .then(() => loggingCallback(null, {'message': `added label ${context.config.label} to ${context.webhook.pull_request.url}`}));
       } else {
-        return githubRequest(addComment(context), clabotToken)
+        const usersWithoutCLA = nonContributors.map(function (contributorId) {
```

Comment:
Actually - just thought about this some more. If the users start creating complicated handlebars templates, how do they test that they are correct?

So, yes, I'm happy with this manipulation! Although, it can be simplified ... arrow functions and template literals :-)

```
const usersWithoutCLA = nonContributors.map(contributorId => `@ ${contributorId}`)
                                                                       .join(', ');
```